freshmaker/handlers/__init__.py

```diff
@@ -434,6 +434,35 @@ def allow_build(self, artifact_type, **criteria):
                        criteria, artifact_type.name.lower())
         return False
 
+    def _mark_event_complete_when_all_builds_done(self, db_event):
+        """Mark Event COMPLETE
+
+        When all builds in an event have been done (either in FAILED or DONE
+        state), the event should be marked as COMPLETE accordingly. If not all
+        finish, nothing change to the state.
```

Comment:
I just have a minor suggestion to rephrase that last sentence in the docstring:
"If not all builds are finished, then the state is not changed"?

Reply:
Addressed, thanks.